invenio_communities/members/records/models.py

```diff
@@ -27,6 +27,7 @@ class MemberModel(db.Model, RecordMetadataBase):
     community_id = db.Column(
         UUIDType,
         db.ForeignKey(CommunityMetadata.id, ondelete="CASCADE"),
+        nullable=False
```

Comment:
It was this little bugger...

Reply:
I was so focused to check the Alembic recipes, that I didn't have a look to the `models`... good job and thanks for fixing it! :)